engine/gamesys/src/gamesys/scripts/script_http.cpp

```diff
@@ -93,7 +93,7 @@ namespace dmGameSystem
      * @param [options] [type:table] optional table with request parameters. Supported entries:
      *
      * - [type:number] `timeout`: timeout in seconds
-     * - [type:string] `path`: path on disc where to download the file. Only overwrites the path if status is 200. [icon:attention] Not available in HTML5 build
+     * - [type:string] `path`: path on disc where to download the file. Only overwrites the path if status is 200. [icon:attention] Path should be absolute
```

Comment:
What will happen if IndexedDB isn't supported?

Reply:
https://emscripten.org/docs/api_reference/Filesystem-API.html#memfs
As I understand if no persistent storage is presented - in-memory FS will be used.